Resources/Prototypes/Entities/Mobs/Player/observer.yml

```diff
@@ -71,6 +86,19 @@
     checkCanInteract: false
     event: !type:ToggleLightingActionEvent
 
+#Frontier action
+- type: entity
+  id: ActionObserverShowRadar
+  name: Mass Scanner Interface
+  description: View a mass scanner interface.
+  components:
+  - type: InstantAction
+    icon: { sprite: Structures/Machines/parts.rsi, state: box_0 }
+    iconOn: Structures/Machines/parts.rsi/box_2.png
+    keywords: [ "AI", "console", "interface" ]
+    checkCanInteract: false
+    event: !type:ToggleIntrinsicUIEvent { key: enum.RadarConsoleUiKey.Key }
+
```

Comment:
Could this not reuse ActionAGhostShowRadar?

Reply:
For some reason i only managed it to work with the checkCanInteract: false